src/EliteChroma.Core/Layers/HyperspaceLayer.cs

```diff
@@ -141,6 +144,11 @@ private void DrawStars(Stars stars, KeyboardCustom keyboard)
                     c = c.Combine(Color.White, t - 4);
                 }
 
+                if (y == 0 && x >= 1 && x < 5)
```

Comment:
Please change this to `if (y == 0 && x >= 1 && x < ChromaLinkConstants.MaxLeds)`.